code/src/index.css

```diff
@@ -11,3 +11,164 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+header {
+  font-size: 35px;
+  font-family: "Roboto Mono", monospace;
+  letter-spacing: 5px;
+  background-color: rgba(219, 120, 120, 0.265);
+  padding: 5vh;
+  border-radius: 40%;
+}
+
+.outer-wrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.inner-wrapper {
+  display: flex;
+  margin: 0 auto;
+  flex-direction: column;
+  justify-content: center;
+  row-gap: 30px;
+  align-items: center;
+  margin-top: 5%;
+  margin-bottom: 10%;
+}
+
+.input-wrapper,
+.list-wrapper {
+  display: flex;
+  flex-direction: column;
+  row-gap: 30px;
+  width: 100%;
+}
+
+.input-container {
+  background-color: rgb(219, 219, 219);
+  box-sizing: border-box;
+  border: 2px solid black;
+  box-shadow: 5px 5px;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  padding: 10px 14px;
+  height: 230px;
+  width: 100%;
+  max-width: 550px;
+  font-family: "Roboto Mono", monospace;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.lable-text {
+  font-weight: 600;
+  font-size: large;
+  padding: 1vh;
+  margin-bottom: 1vh;
+}
+
+.text-input {
+  word-wrap: break-word;
+  padding: 5px;
+  resize: none;
+  height: 8vh;
+}
+
+.counter {
+  margin: 0;
+  margin-left: 1vh;
+  margin-top: 1vh;
+}
+
+.btn-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 1vh;
+  margin-top: 1vh;
+}
+
+.thought-btn {
+  background-color: rgb(223, 157, 157);
+  font-weight: 600;
+  font-size: 15px;
+  border: 0;
+  height: 40px;
+  border-radius: 20px;
+  width: 70%;
+  cursor: pointer;
+}
+
+.list-card {
+  box-sizing: border-box;
+  border: 2px solid black;
+  box-shadow: 5px 5px;
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  padding: 10px 14px;
+  height: 180px;
+  width: 100%;
+  max-width: 550px;
+  font-family: "Roboto Mono", monospace;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+.thought-text {
+  word-wrap: break-word;
+  margin: 2vh;
+}
+
+.like-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  column-gap: 5px;
+  color: rgb(45, 45, 45);
+  font-size: 15px;
+  justify-content: space-between;
+  overflow: hidden;
+}
+
+.like-btn {
+  border-radius: 70%;
+  background-color: rgb(231, 160, 160);
+  border: none;
+  padding: 5px;
+  font-size: 20px;
+  height: 50px;
+  width: 50px;
+  cursor: pointer;
+  transition: all 0.2s ease;
+  margin-left: 10px;
+  margin-right: 5px;
+}
+
+.like-btn:hover {
+  background-color: rgb(238, 123, 123);
```

Comment:
Subtle and nice hovering effect - recommendation would be to add this only on desktop to avoid making the user "double click" when on a mobile/tablet with touch.